### ISSUE TITLE
Autostarts with spell check enabled

### ISSUE THREAD

**Issue by mlnrski** [UberWriter/uberwriter#105]
### ENVIROMENT
- Linux distribution: Debian Buster
- Desktop Enviroment: GNOME Shell
- DE version: 3.30
- GTK version: 3.24.2

App installed via flatpak.

```
Subject: v2.1.4 (a930b969)
Commit: 93cddae302e42846d27a343aae358ebb72d9d4e6a962f171e05e830a7b3113c1
```

### BUG

Starts with autospellcheck enabled even though it has been disabled previously. This looks like a regression of #57

**Comment by somas95** [comment]
In fact we don't have even a gschema set to store the preferences in this regard

**Comment by somas95** [comment]
Addressed at a1a50d48905f83c178f9923f24d7879eb73f9980
